small grammatical change (#325)
datacarpentry/python-ecology-lesson: _episodes/00-before-we-start.md

  - "Demonstrate how to provide sufficient information for troubleshooting with the Python user community."
keypoints:
  - "Python is an open source and platform independent programming language"

>   - "SciPy is an ecosystem for Python provides the tools necessary for scientific computing"

Reviewer: ```suggestion
  - "The SciPy ecosystem provides tools that enable scientific computing in Python."
```
